### PR TITLE
Accessibility features

### PR DESCRIPTION
**Pull request by Addisu87**
We are working as a team and incorporate the following points as a summary.
     -Title improved
     -Red borders are included when forms are not properly filled (invalid)
     -Audio control is included in the first card for text to speech
     -Resize pictures in the work section for hover
     -button color "let's connect" corrected
    -Navigate links configured to move inside the page
    -All the headings were checked, no changes were needed
    -Contrast color checked, no changes needed
    - Heading hierarchy, image alternatives, multimedia, moving content, and page structure - no issue found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -173,14 +179,14 @@ <h2 class="title">Multi-post stories</h2>
             </ul>
             <p class="text">
               Experimental content creation feature that allows users to add to
```
Comment (ShahierNashaat) [inline index.html:179]: - [ ] Please make your webpage resizeable.
Screenshot from 2022-05-03 23-43-18

File: index.html [hunk 2]
```diff
@@ -64,8 +64,14 @@ <h1 class="heading-title">
...
           </p>
 
+          <audio controls>
+            <source src="images/audio.mp3" type="audio/mp3" />
+          </audio>
+
           <div class="social-links">
-            <h3 class="heading">LET'S CONNECT</h3>
+            <a href="#contact">
+              <h4 class="heading-connect">LET'S CONNECT</h4></a
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change the `h4` tag to `h2` tag as the Heading levels should have a meaningful hierarchy, or you can change it to any other tag but not the heading tags.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Addisu87 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @andym80 , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**